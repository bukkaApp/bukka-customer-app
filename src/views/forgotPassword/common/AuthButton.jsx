import React from 'react';

import PropTypes from 'prop-types';

import Button from 'Components/button/Button';

const SubmitButton = ({ ...props }) => (
  <div className="col-11 mx-auto">
    <Button {...props} />
  </div>
);

const AuthButton = props => (
  <SubmitButton
    type="submit"
    id="submit"
    classNames={`col-md-12 primary-button mt-5
    ${!props.isFormCompleted ? 'disabled' : 'button'}
    `}
    text="Send Confirmation"
    key="0"
    handleClick={() => {}}
  />
);

export default AuthButton;

AuthButton.defaultProps = {
  isFormCompleted: false,
  title: '',
};

AuthButton.propTypes = {
  isFormCompleted: PropTypes.bool,
};

SubmitButton.defaultProps = {
  title: ''
};

SubmitButton.propTypes = {
  title: PropTypes.string
};
