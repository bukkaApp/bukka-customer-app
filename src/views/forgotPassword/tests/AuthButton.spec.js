import React from 'react';
import AuthButton from '../common/AuthButton';


describe('AuthButton component', () => {
  const props = {
    isFormCompleted: true,
  };
  const wrapper = mount(<AuthButton {...props} />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
