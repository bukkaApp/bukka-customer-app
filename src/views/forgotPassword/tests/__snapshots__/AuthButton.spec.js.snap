// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthButton component renders properly 1`] = `ReactWrapper {}`;
